packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/funman/FunmanController.java

```diff
@@ -73,18 +77,29 @@ public ResponseEntity<TaskResponse> createValidationRequest(@RequestBody final J
 			taskRequest.setUserId(currentUserService.get().getId());
 			taskRequest.setInput(objectMapper.writeValueAsBytes(input));
 
-			// TODO:
-			// - create Simulation for tracking
-			// - mark Simulation as pending
 			final UUID uuid = UUID.randomUUID();
+			final Simulation sim = new Simulation();
+
+			sim.setId(uuid);
```

Comment:
You shouldn't have to set the UUID like this - after creation it should come back with one